spec/system/user_integrations_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe 'user show page tests', type: :system do
+  test_user = User.create!(name: 'Test User', bio: 'With a bio')
+  post_one = Post.create!(title: 'Test Post 1', text: 'With some text 1', author: test_user)
+  post_two = Post.create!(title: 'Test Post 2', text: 'With some text 2', author: test_user)
+  post_three = Post.create!(title: 'Test Post 3', text: 'With some text 3', author: test_user)
+  post_four = Post.create!(title: 'Test Post 4', text: 'With some text 4', author: test_user)
+  post_five = Post.create!(title: 'Test Post 5', text: 'With some text 5', author: test_user)
+
+  describe 'index page' do
+    it 'shows the right content' do
+      visit users_path
+      expect(page).to have_content('Users')
+    end
+  end
```

Comment:
- [ ] As required for the project, kindly ensure that you have a Spec file which only tests `User` index page.

- [ ] Once you have create that file, you should ensure that, it includes all the following tests:
  * Tests that asserts you can see the username of all other users.
  * Tests that asserts you can see the profile picture for each user.
  * Tests that asserts you can see the number of posts each user has written.
  * Tests that redirects to the user's show page when you click on a user.